spec/features/observer_spec.rb

```diff
@@ -39,7 +39,7 @@
   it "allows a user to add a reason when adding an observer" do
     reason = "is the archbishop of banterbury"
     proposal = create(:proposal)
-    observer = create(:user)
+    observer = create(:user, client_slug: nil)
```

Comment:
that's probably correct. there's lots of tests for `Proposal` w/ no client_data extension. That's good, imo, because its core functionality that applies to all clients.

fwiw the User factory does this:
```
client_slug { ENV['CLIENT_SLUG_DEFAULT'] || 'ncr' }
```